tensorflow/core/platform/s3/BUILD

```diff
@@ -81,7 +81,12 @@ cc_library(
     hdrs = [
         "s3_file_system.h",
     ],
-    deps = [
+    deps = select({
+        "@org_tensorflow//tensorflow:windows": [
+            "//tensorflow/core/platform:retrying_file_system"
+        ],
+        "//conditions:default": [],
+    }) + [
```

Comment:
I'm wondering why we only need the retrying file system only on windows. Is it because of linking issues?

Reply:
Yes. It seems that `retrying_file_system` can not link into s3 dynamically